### PR TITLE
Complete does_each_zone_have_only_one_terminal

### PR DESCRIPTION
**Pull request by weilixu**
One logic issue relate to this function is [posted](https://github.com/pnnl/ruleset-checking-tool/issues/627). Waiting for RDS team response

### CODE REVIEW

File: rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/does_each_zone_have_only_one_terminal.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    bool
+        True: each zone input to this function only has one terminal unit or no terminal
+        False: any zone has more than one terminal
+    """
+    does_each_zone_have_only_one_terminal_flag = True
+    for zone_id in zone_id_list:
+        zone = find_exactly_one_with_field_value(
+            "$.buildings[*].building_segments[*].zones", "id", zone_id, rmi_b
+        )
+        if zone.get("terminals") is not None and len(zone["terminals"]) > 1:
```
Comment (yunjoonjung-PNNL) [inline rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/does_each_zone_have_only_one_terminal.py:22]: Could this logic also determine ```no terminal units with the zone``` case?

**Review by yunjoonjung-PNNL — APPROVED** [review]
Approved. I'd appreciate it if you could double-check the comment.

File: rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/does_each_zone_have_only_one_terminal.py (hunk 1 above)
Comment (weilixu) [reply rct229/ruleset_functions/baseline_systems/baseline_hvac_sub_functions/does_each_zone_have_only_one_terminal.py:22]: It is possible. I posted this issue [here ](https://github.com/pnnl/ruleset-checking-tool/issues/627)for the `no terminal` situation

**Review by charliepnnl — APPROVED** [review]
Approved contingent on the response from the RDS team.
A more appropriate name for this function, as currently defined by the RDS, would be _does_each_zone_have_at_most_one_terminal_.

**Comment by weilixu** [comment]
> Approved contingent on the response from the RDS team. A more appropriate name for this function, as currently defined by the RDS, would be _does_each_zone_have_at_most_one_terminal_.

I have discussed the issue with RDS, they agree that this shall be only one terminal (null, 0 or more than 1 shall return false.) I will update the RDS language.

**Pull request merged**